includes/block-patterns/class-sensei-block-patterns.php

```diff
@@ -118,6 +118,14 @@ public function register_block_patterns( $current_screen ) {
 				require __DIR__ . "/{$post_type}/{$block_pattern}.php"
 			);
 		}
+
+		if ( Sensei()->feature_flags->is_enabled( 'learning_mode_v2' ) ) {
```

Comment:
We can extract it to a separate module, focused on loading the learning mode patterns.